cypress/e2e/specialFeatures/atiAnalytics/index.cy.js

```diff
@@ -472,7 +472,7 @@ const ampTestSuites = canonicalTestSuites.filter(supportsAmp).map(testSuite => {
   return {
     ...testSuite,
     path: getPathWithSuffix({ path: testSuite.path, suffix: '.amp' }),
-    useReverb: false,
+    useReverb: true,
```

Comment:
`useReverb: true,` is set here to align with the [AmpATIAnalytics](https://github.com/bbc/simorgh/blob/5bc5e9831bc38b2b95ec68e212dc89faf589033d/src/app/components/ATIAnalytics/amp/ampAnalyticsJson.ts#L13) implementation in which,
the `reverbParams` object is created and passed to function, `getAmpAnalyticsPageViewUrl`, of reverb-url-helper when `useReverb` is set to `true` in the service configuration.